src/lib.rs

```diff
@@ -324,6 +326,7 @@ pub(crate) unsafe fn rav1d_open(c_out: &mut *mut Rav1dContext, s: &Rav1dSettings
     let mut n: c_uint = 0 as c_int as c_uint;
     while n < (*c).n_fc {
         let f: &mut Rav1dFrameData = &mut *((*c).fc).offset(n as isize);
+        addr_of_mut!(f.frame_thread).write(Default::default());
```

Comment:
Why did you add this? This is to initialize the field the first time right (hence using `write` instead of writing to the field normally)? Adding this makes sense if we weren't doing it already, but was there a reason you needed to add this in this PR? I don't see how your changes would have made this necessary in a way that it wasn't before.

Reply:
Ah nevermind, I reread the PR description and I see you mentioned this.